cogs/commands.py

```diff
@@ -366,7 +366,15 @@ async def pipsearch(self, ctx, term, order: lambda string: string.lower() = 'rel
                 em = discord.Embed(title=f"Searched for {term}", description=f"No [results]({search}) found.")
                 em.colour = discord.Colour.red()
             await ctx.send(embed=em)
-    
+            
+    @commands.command(name="codeblock")
+    async def cb(self, ctx, *, text):
+        """Puts your code into a copyable codeblock"""
+        await ctx.message.delete()
+        em = discord.Embed(description=f'>>> \```\n{text}\n```')
```

Comment:
Dont do an embed imo because people on mobile cant copy paste from embeds

Reply:
Code will also frequently switch to a new line due to the limited with of the embed.